src/app/team-membership-dialog/team-membership-dialog.component.less

```diff
@@ -26,28 +26,26 @@
     position: absolute;
     list-style: none;
     padding: 0;
-    border: 1px solid $color-pf-black-400;
-    background: $color-pf-white;
+    border: 1px solid;
+    border-color: @color-pf-black-400;
+    background: @color-pf-white;
     li {
-      width: $width100;
+      .width-100;
```

Comment:
I believe this should just be 100%. The width-100 class is really only used a shortcut used in the HTML pages.